apps/devhub/views.py

```diff
@@ -677,8 +679,15 @@ def standalone_upload_detail(request, uuid):
 @dev_required
 @json_view
 def upload_detail_for_addon(request, addon_id, addon, uuid):
-    upload = get_object_or_404(FileUpload, uuid=uuid)
-    return json_upload_detail(request, upload, addon_slug=addon.slug)
+    with statsd.timer('devhub.upload_detail_for_addon'):
```

Comment:
Every view is timed already as well. This one is at https://app.datadoghq.com/metric/explorer?live=true&page=0&is_auto=false&from_ts=1452542602739&to_ts=1452629002739&tile_size=m&exp_metric=addons_prod.view.devhub.views.upload_detail_for_addon.GET.avg&exp_scope=&exp_agg=avg&exp_row_type=metric